src/main/java/com/example/shorturl/controller/login/LogInController.java

```diff
@@ -0,0 +1,60 @@
+package com.example.shorturl.controller.login;
+
+import com.example.shorturl.model.roles.RoleEntity;
+import com.example.shorturl.model.user.UserEntity;
+import com.example.shorturl.model.user.UserRequest;
+import com.example.shorturl.model.user.UserResponse;
+import com.example.shorturl.service.user.UserService;
+import com.example.shorturl.utils.exceptions.UserException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+
+import javax.validation.Valid;
+import java.util.Objects;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("api/auth")
+public class LogInController {
```

Comment:
Let's name it AuthController, because it's not only for login :)

Reply:
Fixed